src/modules/dashboard/Dashboard.constants.tsx

```diff
@@ -1,13 +1,65 @@
-//Components
-import { PushAlpha, PushBot, PushDev } from 'blocks';
+import { Box, PushAlpha, PushBot, PushDev, TabItem } from 'blocks';
+import { TrendingChannelsList } from './components/TrendingChannelsList';
+import { HottestChannelsList } from './components/HottestChannelsList';
+import { SubscribedChannelsList } from './components/SubscribedChannelsList';
 
-//Types
-import { ChatType, DashboardChannelTabsType, EnvKeys, SourceKeys } from './Dashboard.types';
+import { ChatType, EnvKeys, SourceKeys } from './Dashboard.types';
 
-export const dahboardChannelTabs: DashboardChannelTabsType = [
-  { label: 'Trending Channels', value: 'trending' },
-  { label: 'Hottest Channels', value: 'hottest' },
-  { label: 'Subscribed', value: 'subscribed' },
+export const dashboardChannelTabs: TabItem[] = [
```

Comment:
This needs to be inside component. 

Reason: If I need to pass some props to the tab component itself, I cannot do this with this approach.